src/store/search-store.js

```diff
@@ -107,7 +108,7 @@ const actions = ImageService => ({
         );
       })
       .catch((error) => {
-        commit(FETCH_IMAGES_ERROR);
+        commit(FETCH_IMAGES_ERROR, { errorMessage: error.message });
```

Comment:
Usually the `message` that comes in the API response is something a bit too technical, for example `Error fetching images: Request failed with status code 500`. I think it's worth checking the status code (500, 404 are the most common) and providing a more user friendly error message